Sources/TokamakCore/Fiber/Fiber.swift

```diff
@@ -83,22 +85,33 @@ public extension FiberReconciler {
     /// Parent references are `unowned` (as opposed to `weak`)
     /// because the parent will always exist if a child does.
     /// If the parent is released, the child is released with it.
-    unowned var parent: Fiber?
+    @_spi(TokamakCore)
+    public unowned var parent: Fiber?
 
     /// The nearest parent that can be mounted on.
     unowned var elementParent: Fiber?
 
+    /// The nearest parent that receives preferences.
+    unowned var preferenceParent: Fiber?
+
     /// The cached type information for the underlying `View`.
     var typeInfo: TypeInfo?
 
     /// Boxes that store `State` data.
     var state: [PropertyInfo: MutableStorage] = [:]
 
+    /// Subscribed `Cancellable`s
```

Comment:
nit: please expand these doc comment to explain why `PropertyInfo` is used as a dictionary key